kubetest/extract_k8s.go

```diff
@@ -39,6 +39,7 @@ type extractMode int
 const (
 	none    extractMode = iota
 	local               // local
+	git                 // A git tag version from the repository (e.g. v1.13.7-gke.29)
```

Comment:
is this only a local clone then? it also doesn't seem to fetch tags etc.?

why would we use this over --local and doing a `git clone <k8s-fork> && git checkout vfoo` first?

Reply:
The idea is to leverage the *version* from git tag, using the same method as `hack/print-workspace-status.sh`, and use that to determine the built kubernetes version to `extract`.